### ISSUE TITLE
No toolbar button for Page Info

### ISSUE THREAD

**Issue by sergwish** [Quicksaver/OmniSidebar#78]
Please, add a toolbar button for Page Info like you did for Downloads. It is inconvenient to turn on the sidebar menu dropdown just to be able to access Page Info.

**Comment by Quicksaver** [comment]
I'll add one, there's no harm in doing it anyway. But you can always enable the "Always open in the sidebar -> Page Info" preference, and right-click on the page and choose "View Page Info".

**Comment by sergwish** [comment]
Depending on the page element you right-click on, the "Page Info" line may be present or not. Same with navigation options. Or right-click event may be processed by page element itself instead of opening context menu.

In general, I find Firefox context menu inconvenient: it is not configurable, inconsistent, lacks distinguishing icons and often contains rarely-if-ever-used commands. 

I think, Firefox should do context menu the same way you did context menu for Sidebar toolbar button - configurable and with icons on top.

**Comment by Quicksaver** [comment]
I haven't added this yet (I still will, just don't really know when) but I just realised there's another way you can do this. Enable "Show sidebar title" and "Sidebar menu dropdown in title". This way, when you right-click the OmniSidebar main toolbar button, in the menu that appears there should always be the "View Page Info" entry in there.

**Comment by sergwish** [comment]
That's true. But clicking "View Page Info" when it is already checked will close the sidebar. BTW, with sidebar hidden, it does not open when I select View Page Info from any context menu (page or sidebar button). I need to open the sidebar to see the Page Info collected.

There is also a bug: data in Page Info is not updated when sidebar is opened from another tab or after tab reload. This can misinform the user.

**Comment by Quicksaver** [comment]
> BTW, with sidebar hidden, it does not open when I select View Page Info from any context menu (page or sidebar button). I need to open the sidebar to see the Page Info collected.

Works for me. Does this also happen if you disable all your other add-ons except for OSB? If so, can you give me the exact steps you take that leads to this?

> There is also a bug: data in Page Info is not updated when sidebar is opened from another tab or after tab reload. This can misinform the user.

It's not really a bug, it's by design actually. The same is also true for the normal Page Info dialog, it doesn't update itself, and neither should the sidebar in this case, it should be subject to the same rules/behavior criteria.
